src/frontend/admin/src/pages/_app.tsx

```diff
@@ -25,6 +29,21 @@ interface MyAppProps extends AppProps {
 const clientSideEmotionCache = createEmotionCache();
 
 export default function App({ Component, pageProps }: MyAppProps) {
+  const [queryClient] = useState(
+    () =>
+      new QueryClient({
+        defaultOptions: {
+          queries: {
+            cacheTime: REACT_QUERY_SETTINGS.cacheTime,
+            staleTime: 60 * 1000, // 1 mi
```

Comment:
Why not use `REACT_QUERY_SETTINGS.staleTime.default` ?

Reply:
already done